ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -65,15 +66,15 @@ ${SCRIPT_BREW_TAG}=    ${EMPTY}    # ^[0-9]+$
 Verify External Dependency Operators Can Be Deployed
     [Documentation]    Checks the pre-required Operators can be installed
     ...                and configured
-    [Tags]    ODS-2326    WatsonX
+    [Tags]    ODS-2326
```

Comment:
Missing TierX tag

Reply:
@bdattoma  What is the status of this? Is it okay or shall it be added?

Reply:
just updated the PR